orchestra/static/orchestra/timing/timecard/controller.js

```diff
@@ -0,0 +1,51 @@
+(function() {
+  'use strict';
+
+  angular.module('orchestra.timing')
+    .controller('TimecardController', function($routeParams, $scope, timeEntries) {
+      var vm = this;
+      vm.taskId = $routeParams.taskId;
+
+      vm.timeEntries = timeEntries;
+
+      vm.datetimeFromKey = function(entries) {
+        if (entries.$key) {
+          return moment(entries.$key);
+        }
+      };
+
+      vm.humanizeDuration = function(duration) {
+        return duration.get('hours') + 'h ' + duration.get('minutes') + 'm';
```

Comment:
this appears to truncate, rather than round up.

can we round up consistently throughout?